README.md

```diff
@@ -7,8 +7,58 @@ Primarily, it allows reading a service description and generating various
 types of output.  It uses the [Jinja2 templating language](https://jinja.palletsprojects.com) ([(alt. link)](https://jinja2docs.readthedocs.io/en/stable/))
 for most output definitions.
 
-# Testing it out
+## Getting started
 
+### Prerequisites
+* Python 3.7 installed
```

Comment:
3.8

Reply:
Right, I've made a force push on the branch to solve this. Should be ok now